### PR TITLE
Add News Utility Regression Tests

### PR DESCRIPTION
**Pull request by LorenzoMinto**

### CODE REVIEW

File: modules/web-discovery-project/tests/integration/utility-regression-test.es [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+    describe("UtilityRegression tests", () => {
+        const getSuffix = (path = "base") => `/${path}`;
+        const getUrl = (path = "base") => testServer.getBaseUrl(getSuffix(path));
+
+        const WebDiscoveryProject = app.modules["web-discovery-project"].background.webDiscoveryProject;
+        const pipeline = app.modules["webrequest-pipeline"].background;
+        let addPipeline;
+
+        const addCookiesToRequest = (request, response) => {
+          response.modifyHeader("Cookie", "cookie-text")
```
Comment (remusao) [inline modules/web-discovery-project/tests/integration/utility-regression-test.es:67]: Did we need to add specific cookies here for consent popups?

**Review by remusao — APPROVED** [review]

File: modules/web-discovery-project/tests/integration/utility-regression-test.es [hunk 2]
```diff
@@ -148,7 +148,7 @@ export default () => {
                 WebDiscoveryProject.isAlreadyMarkedPrivate(url, (res) => {
                   expect(res.private, "url is marked as private!").equal(0);
                 });
-              }, 3);
+              });
```
Comment (LorenzoMinto) [inline modules/web-discovery-project/tests/integration/utility-regression-test.es:151]: we don't want to retry if a test fails?

File: modules/web-discovery-project/tests/integration/utility-regression-test.es [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+                  if (canonical_url != null) {
+                    return (await new Promise((resolve) => WebDiscoveryProject.db.getURL(canonical_url, resolve))).length == 1
+                  }
+                });
+                await WebDiscoveryProject.forceDoubleFetch(url);
+                await waitFor(async () => (await new Promise((resolve) => WebDiscoveryProject.db.getURL(url, resolve))).length == 0);
+                WebDiscoveryProject.isAlreadyMarkedPrivate(url, (res) => {
+                  expect(res.private, "url is marked as private!").equal(0);
+                });
+              });
```
Comment (remusao) [inline modules/web-discovery-project/tests/integration/utility-regression-test.es:151]:
```suggestion
              }, 3);
```

File: modules/web-discovery-project/tests/integration/utility-regression-test.es (hunk 1 above)
Comment (LorenzoMinto) [reply modules/web-discovery-project/tests/integration/utility-regression-test.es]: removed for now 👍

**Review by remusao — APPROVED** [review]

**Pull request merged**